internal/ioctl/ioctl.go

```diff
@@ -172,6 +264,11 @@ func AddDeviceToTracking(device types.DevID) error {
 		minor: C.int(device.Minor),
 	}
 
+	err = setMissingKernelEntries(dev)
```

Comment:
This does not need to be called every time you add a device to tracking. This can be moved to the ```main()``` function and can be called once before starting the workers.